### PR TITLE
Fix for maven warnings

### PR DESCRIPTION
**Pull request by mohanaraosv**
Resolve maven warning

image

Pull request description

- Removed duplicated dependency declaration
- Updated `junit-jupiter-api` junit dependency co-ordinates 

> For detailed contributing instructions see https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

### CODE REVIEW

File: page-object/pom.xml [hunk 1]
```diff
@@ -39,8 +39,7 @@
     </dependency>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter</artifactId>
-      <version>RELEASE</version>
+      <artifactId>junit-jupiter-api</artifactId>
       <scope>test</scope>
     </dependency>
```
Comment (robertvolkmann) [inline page-object/pom.xml:44]: You can remove the whole dependency because it is not used anymore.

Comment (mohanaraosv) [reply page-object/pom.xml]: Nice catch 👍

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Thanks for chasing out these warnings!

@all-contributors please add @mohanaraosv for code

**Comment by iluwatar** [comment]
@all-contributors please add @robertvolkmann for review

**Pull request merged**